tasks/libs/common/git.py

```diff
@@ -270,3 +275,20 @@ def get_last_release_tag(ctx, repo, pattern):
             last_tag_name = last_tag_name_with_suffix.removesuffix("^{}")
     last_tag_name = last_tag_name.removeprefix("refs/tags/")
     return last_tag_commit, last_tag_name
+
+
+def get_git_config(key):
+    result = subprocess.run(['git', 'config', '--get', key], capture_output=True, text=True)
+    return result.stdout.strip() if result.returncode == 0 else None
+
+
+def set_git_config(key, value):
+    subprocess.run(['git', 'config', key, value])
```

Comment:
These helpers are a good idea! They could even be split in their own PR and be used throughout our tasks if this PR takes long to be merged (but definitely not required)

Reply:
I will make a ticket to update the rest of our tasks